.github/workflows/ci.yml

```diff
@@ -60,6 +60,19 @@ jobs:
             exit 0
           fi
 
+  doxygen:
+    runs-on: ubuntu-20.04
```

Comment:
Does this need to be ubuntu-20.05 instead of ubuntu-latest?

Reply:
It does. It cannot work as is with the ubuntu latest. See the yml files in [MQTT](https://github.com/FreeRTOS/coreMQTT/blob/master/.github/workflows/ci.yml#L53-L64) and [HTTP](https://github.com/FreeRTOS/coreHTTP/blob/master/.github/workflows/ci.yml#L53-L65) repo.